### DISCUSSION TITLE
New Feature Request: Tag

### DISCUSSION THREAD

**Discussion by CRC32EX** [primefaces/community#549] [PrimeReact] [answered]
![image](https://github.com/primefaces/community/assets/25629617/7294a664-19ea-41f3-a2bf-30f070e6eeed)

- https://primereact.org/tag/
- https://primereact.org/button/

# Request
`Tag` supports 5 colors.
`Button` supports 7 colors.

So, Please add new color `Secondary`, `Help` to Tag.

**Comment by melloware** [comment, marked as answer]
Ticket opened: https://github.com/primefaces/primereact/issues/5404
